### PR TITLE
fix: resolve panic on startup due to duplicate language flag declaration

### LINKED ISSUE #30: Panic on startup
Running `go run ./cmd/generator`:

```text
configure flag redefined: language
panic: configure flag redefined: language
```

### PR DESCRIPTION
**Pull request by burkedavison**
Resolves #30

### CODE REVIEW

File: internal/command/flags.go [hunk 1]
```diff
@@ -43,8 +43,7 @@ func addFlagBranch(fs *flag.FlagSet) {
 }
 
 func addFlagLanguage(fs *flag.FlagSet) {
-	fs.StringVar(&flagLanguage, "language", "", "language to generate code for")
```
Comment (julieqiu) [inline internal/command/flags.go:46]: oops =P

I actually meant to keep this line and delete the other one

File: internal/command/flags.go [hunk 2]
```diff
@@ -31,26 +30,19 @@ var (
 )
 
 func addFlagAPIRoot(fs *flag.FlagSet) {
-	fs.StringVar(&flagAPIRoot, "api-root", "", "location of googleapis repository")
+	fs.StringVar(&flagAPIRoot, "api-root", "", "(Required) location of googleapis repository")
```
Comment (julieqiu) [inline internal/command/flags.go:33]: I don't _think_ this is required (see @jskeet's comments on https://github.com/googleapis/generator/pull/25/files)

Comment (julieqiu) [inline internal/command/flags.go:33]: I don't _think_ this is required (if flagAPIRoot is empty, we clone googleapis)

https://github.com/googleapis/generator/pull/25/files#diff-69d3575d337e6861d23e877e04bbdb99f437a82661f706f055a50e82b1e8282aR81

I think this needs to be fixed:

https://github.com/googleapis/generator/pull/25/files#diff-69d3575d337e6861d23e877e04bbdb99f437a82661f706f055a50e82b1e8282aR54

**Review by julieqiu — APPROVED** [review]

File: internal/command/flags.go (hunk 2 above)
Comment (burkedavison) [reply internal/command/flags.go]: It does behave this way for generate -- I was playing around and found it required for the 'configure' command, but then it's also not yet implemented.

I've updated to reflect the generate command's behavior which is the current focus.

**Pull request merged**